src/classes/UTIL_Describe.cls

```diff
@@ -680,6 +680,30 @@ public class UTIL_Describe {
         }
     }
 
+    /*******************************************************************************************************
+     * @description Returns the ID of the custom Administrative Account record type
+     * @return The ID of the custom Administrative Account record type.
+     */
+    public static Id getCustomAdminAccRecTypeId() {
+        Id customAdminRecTypeId = Util_CustomSettingsFacade.getSettings().Administrative_Account_Record_Type__c;
+        if (getRecTypesMapByDevName('Account').values().contains(customAdminRecTypeId) == false) {
+            throw new ERR_ExceptionHandler.CustomAdminRecordTypelMissingException(Label.customAccountRecordTypeMissingError);
+        }
+    }
+
+    /*******************************************************************************************************
+     * @description Returns the ID of the default Administrative Account record type
+     * - If no default account record type is found, throw exception.
+     * @return The ID of the default Administrative Account record type.
+     */
+    public static Id getDefaultAdminAccRecTypeId() { 
+        Id defaultAdminRecTypeId = getRecTypesMapByDevName('Account').get('Administrative');
+        if (defaultAdminRecTypeId == null) {
+            throw new ERR_ExceptionHandler.DefaultAdminRecordTypelMissingException(Label.defaultAccountRecordTypeMissingError);
```

Comment:
@xinnanlisf @MatthewBlanski  I know there's still some work going on here...but a heads up if a customer does not have a record type named "Administrative" (e.g. they deleted it because they use Household only or they renamed it "Administrative_Account", most unit tests that reference UTIL_CustomSettingsFacade.getSettings are going to fail.

Reply:
@kevinswiggum thanks for the input! That has been one of our biggest worries too. Ideally this should only run for fresh installs and we are preventing customers from installing eda if default admin record type (aka 'Administrative' does not exist already. Also, the custom settings for unit tests in EDA should have been isolated from the actual custom setting setup code. We are working on this and hopefully will have a good solution. @MatthewBlanski

Reply:
@xinnanlisf Please add a comment here about only using this in a fresh install context to prevent customer breakage. I don't want this method to leak into other places and to leave a guidepost as to exactly why this exists. Optionally, you may want to change the name of this method to make that even more clear.